src/client/table_client.rs

```diff
@@ -2502,6 +2502,16 @@ impl ObTableAggregation {
         self
     }
 
+    #[inline]
+    pub fn index_name(mut self, index_name: &str) -> Self
+    where
+        Self: Sized,
+    {
+        //agg secondary index support
```

Comment:
```suggestion
        //agg local index support
```
I suppose 'local' is a better name in this area.